tools/backups2datalad/adataset.py

```diff
@@ -210,7 +211,28 @@ async def add(self, path: str) -> None:
 
     async def remove(self, path: str) -> None:
         # `path` must be relative to the root of the dataset
-        await self.call_git("rm", "-f", "--ignore-unmatch", "--", path)
+        try:
+            await self.call_git(
+                "rm",
+                "-f",
+                "--ignore-unmatch",
+                "--",
+                path,
+                stdout=subprocess.PIPE,
+                stderr=subprocess.STDOUT,
+            )
+        except subprocess.CalledProcessError as e:
+            lockfile = self.pathobj / ".git" / "lock"
```

Comment:
that was wrong -- should have been `index.lock`... will commit that fix locally but also will just incorporate precommit